### PR TITLE
Fixes #22422: port of ticket 22188 in 7.2

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/22422

supervised and unsupervised targets were inverted

### CODE REVIEW

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR rebased

File: change-validation/src/main/scala/bootstrap/rudder/plugin/ChangeValidationConf.scala [hunk 1]
```diff
@@ -93,19 +93,22 @@ import com.normation.rudder.services.workflows.NodeGroupChangeRequest
...
-    validationWorkflowService: TwoValidationStepsWorkflowServiceImpl,
-    validationNeeded:          Seq[ValidationNeeded],
-    workflowEnabledByUser:     () => Box[Boolean]
+    status                   : PluginStatus
+  , defaultWorkflowService   : WorkflowService
+  , validationWorkflowService: TwoValidationStepsWorkflowServiceImpl
+  , validationNeeded         : Seq[ValidationNeeded]
+  , workflowEnabledByUser    : () => Box[Boolean]
+  , validatedUserRepo: RoValidatedUserRepository
 ) extends WorkflowLevelService {
```
Comment (fanf) [inline change-validation/src/main/scala/bootstrap/rudder/plugin/ChangeValidationConf.scala:112]: you must not use that syntax anymore, it is not auto-formater friendly. Please keep the other one

**Review by fanf — COMMENTED** [review]
Business logic is ok, it does what it says it do. 
You need to:
- correct syntax (see comments and use `mvn spotless:apply` if you don't have it configured in your editor)
- rebase & correct the commit message, only keep one line with the `Fixes:...`, other seems useless
- comment the PR for what are it's major code changes.

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR rebased

**Pull request merged**